.eslintrc.js

```diff
@@ -3,20 +3,29 @@ module.exports = {
     browser: true,
     es2021: true,
   },
-  extends: ['eslint:recommended', 'plugin:react/recommended'],
+  extends: [
+    'eslint:recommended',
+    'plugin:import/recommended',
+    'plugin:jsx-a11y/recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+  ],
   parserOptions: {
     ecmaFeatures: {
       jsx: true,
     },
     ecmaVersion: 13,
     sourceType: 'module',
   },
-  plugins: ['react'],
```

Comment:
i realized this isn't necessary for our plugins because we're always extending the recommended ruleset, so adding the plugins here was duplicative. it doesn't hurt to have both though, so let me know if you have thoughts on this